### PR TITLE
mz-eb-navigate-in-app

### PR DESCRIPTION
**Pull request by mindyzwan**
**Week 1 -** 
Acceptance Criteria: 
- react-router-dom has been added as a project dependency
- Click a link within the app to show a different view in the app
- When a link is clicked, the browser URL updates to represent the current view

Tests are configured for each and we had no failures for the accessibility test. 

@michellejae & @ClaudiaStar - this is ready for your review! Let us know if you see anything we can improve on :)

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,26 +1,45 @@
 import React from 'react';
 import logo from './logo.svg';
```
Comment (ClaudiaStar) [inline src/App.js]: remove logo import - not used.

File: src/App.js [hunk 2]
```diff
@@ -1,26 +1,45 @@
...
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
-  );
+    <Router>
```
Comment (ClaudiaStar) [inline src/App.js]: Functionality wise - everything looks great and works! Our feedback is that the router itself should be a separate component imported into App. js.

File: src/App.js [hunk 3]
```diff
@@ -1,26 +1,45 @@
...
+          <List />
+        </Route>
+        <Route exact path="/add">
+          <Add />
+        </Route>
+      </Switch>
+    </Router>
+  )
+}
+
```
Comment (ClaudiaStar) [inline src/App.js]: We think each of these functions should be separate components imported into the Router component.

**Comment by gunderodd** [comment]
I updated the branch after making the changes @ClaudiaStar suggested. I think that 1. deleting unnecessary imports is definitely a good call. And 2. moving the router to a new component seems good, to me, but that might be more up for debate; I left the code commented out for now, I figured I'd wait for @mindyzwan 's input before making it permanent. Does this look good to everyone?

File: src/Add.js [hunk 4]
```diff
@@ -0,0 +1,9 @@
+import React from 'react'
+
+function Add() {
+	return (
+			<h1>Add an item to your list here</h1>
+	)
+}
+
+export default Add;
```
Comment (segdeha) [inline src/Add.js]: image

See the red icons at the end of the line? This is GitHub complaining that the file doesn't end with a newline (`\n`). **No need to fix this,** but just be aware that in some systems/languages, that lack of an extra return can cause unintended errors. It's a good habit to be in to always add one last return.

Comment (segdeha) [reply src/Add.js]: Wait, did y'all fix this while I was adding the comment? I don't see the red icons anymore 🤔

Comment (segdeha) [reply src/Add.js]: Oh. It was just GitHub not showing the icons in the other view. Sorry for the comment churn!

File: src/Home.js [hunk 5]
```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+
+function Home() {
+	return (
+			<h1>congrats this is the home page</h1>
```
Comment (segdeha) [inline src/Home.js]: This formatting is a little funky, but not a blocking problem. Consider dedenting (is that a word?) so it's only indented once compared to line 4

File: src/List.js [hunk 6]
```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+
+function List() {
+	return (
+			<h1>This is the list page!</h1>
```
Comment (segdeha) [inline src/List.js]: This formatting is a little funky, but not a blocking problem. Consider dedenting (is that a word?) so it's only indented once compared to line 4

**Pull request merged**